### DISCUSSION TITLE
为什么我按装好了无法使用？

### DISCUSSION THREAD

**Discussion by michael-hongy** [Sanster/IOPaint#204] [Q&A]
我非常喜欢这个项目。我安装好了0.3.4版本的lama cleaner，模型是sd1.5，没有独立显卡。无法连接国际互联网。安装好后，我打开使用界面，选择要去除的部分之后就没反应了。没有去除不需要的部分。请大家帮忙解决一下？谢谢！![mmexport1675847918755.png](https://user-images.githubusercontent.com/84266103/217488014-1c474186-2e35-4248-85cf-2c19a15e958e.png)![mmexport1675847914198.png](https://user-images.githubusercontent.com/84266103/217488070-bc6dcd3a-d509-4bc2-b6d1-d0fd4315677e.png)

**Comment by Sanster** [comment]
sd 模型 cpu 跑不动的，你只是想要擦除物体的话，用 lama 模型就好了

**Reply by michael-hongy** [reply]
请问怎么更换模型呢？我在图形操作界面无法下载模型。手动下载的模型放在哪个路径呢？

**Reply by Sanster** [reply]
https://lama-cleaner-docs.vercel.app/frequently_asked_questions#where-are-models-downloaded
